examples/run/alpaka/full_chain_algorithm.hpp

```diff
@@ -99,15 +99,11 @@ class full_chain_algorithm
                          host_detector_type* detector,
                          std::unique_ptr<const traccc::Logger> logger);
 
-    /// Copy constructor
-    ///
-    /// An explicit copy constructor is necessary because in the MT tests
-    /// we do want to copy such objects, but a default copy-constructor can
-    /// not be generated for them.
-    ///
-    /// @param parent The parent algorithm chain to copy
-    ///
-    full_chain_algorithm(const full_chain_algorithm& parent);
+    full_chain_algorithm() = delete;
+    full_chain_algorithm(const full_chain_algorithm& other) = delete;
+    full_chain_algorithm(full_chain_algorithm&& other) noexcept;
+    full_chain_algorithm& operator=(const full_chain_algorithm& other) = delete;
+    full_chain_algorithm& operator=(full_chain_algorithm&& other) = delete;
```

Comment:
I still feel fairly strongly about this, that if a class is allowed to be move-constructed, then it should be allowed to be moved-assigned as well. I don't like introducing such a non-intuitive behaviour.